pandas/tests/series/indexing/test_datetime.py

```diff
@@ -576,6 +576,30 @@ def test_indexing_over_size_cutoff():
         _index._SIZE_CUTOFF = old_cutoff
 
 
+def test_indexing_over_size_cutoff_period_index():
+
```

Comment:
can you add a comment here (the title is pretty good though) with the issue number as a comment as well